backend/match4everyone/settings/common.py

```diff
@@ -167,8 +168,11 @@
 
 STATIC_URL = "/static/"
 STATIC_ROOT = os.path.join(RUN_DIR, "static")
-STATICFILES_DIRS = (os.path.join(BASE_DIR, "static"),)
 STATICFILES_STORAGE = "whitenoise.storage.CompressedManifestStaticFilesStorage"
+STATICFILES_DIRS = (
+    os.path.join(BASE_DIR, "static"),
+    os.path.normpath(os.path.join(os.path.join(os.path.dirname(BASE_DIR), "frontend"), "dist")),
```

Comment:
This should actually also work as such:
```suggestion
    os.path.normpath(os.path.join(os.path.dirname(BASE_DIR), "frontend", "dist")),
```
And since we never handle paths manually, the `normpath` should not be necessary either or am I missing something? Isn't the path library os agnostic?

Reply:
I read the docs, you should be right